lib/ProductOpener/Display.pm

```diff
@@ -6090,7 +6090,13 @@ $options{favicons}
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.10/css/select2.min.css" integrity="sha256-FdatTf20PQr/rWg+cAKfl6j4/IY3oohFAJ7gVC3M34E=" crossorigin="anonymous">
 <link rel="search" href="$formatted_subdomain/cgi/opensearch.pl" type="application/opensearchdescription+xml" title="$Lang{site_name}{$lang}">
 $header
+<!-- Dark Mode -->
+<link rel="stylesheet" href="$static_subdomain/css/darkMode.css">
```

Comment:
This basically makes dark mode the default - Might introduce unwanted behaviour if JavaScript is disabled. cc @stephanegigandet

Reply:
Dark mode should not be the default.

Reply:
I didn't test in all browsers but in Chromium adding `disabled="true"` to `link` disables stylesheet. I think this should work in other browsers as well. JS code of this PR is based on `disabled` property and [MDN says](https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet/disabled) it's supported everywhere except IE (unknown)

Reply:
> 
> 
> I didn't test in all browsers but in Chromium adding `disabled="true"` to `link` disables stylesheet. I think this should work in other browsers as well. JS code of this PR is based on `disabled` property and [MDN says](https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet/disabled) it's supported everywhere except IE (support is unknown)

Right, but there are some extensions that completely disable Javascript execution. In that case dark mode would be the default. Consider adding `disabled="true"` here and changing it in JS.